partials/schema-prop.html

```diff
@@ -80,9 +81,8 @@
           </div>
         </div>
 
-        {% if prop.description() %}
-          <div class="text-sm markdown">{{ prop.description() | markdown2html | safe }}</div>
-        {% endif %}
+        {{ getDescription(altDescription or prop.description()) }}
```

Comment:
above this line add something like `{{ altDescription | log}}` and tell me what you see in logs in termina